### ISSUE TITLE
Add preferred netblock filter for hosts

### ISSUE THREAD

**Issue by adrianreber** [fedora-infra/mirrormanager2#148]
In the original mirrormanager it was not possible to add netblocks larger than /16 to a host. This filter does not seem to exist anymore. Right now it seems possible to add RFC1918 networks (which does not make much sense in most cases (except Fedora internal infrastructure systems)) and even 0.0.0.0/0.

Looking at the database we have lot's of private networks added as preferred netblocks to different hosts.

See also: https://fedorahosted.org/fedora-infrastructure/ticket/5016

**Comment by nirik** [comment]
How does mm get the client IP? just from the http header or url right? 

if so, the only way a RFC1918 entry could be useful would be: 

a) you pass ip= on your link
or
b) you are in Fedora Infrastructure and you hit proxy01/proxy10 on their internal 10.x.x.x ip's 

right?

**Comment by adrianreber** [comment]
Right, even though people have entered it into the database in does not have much effect. Fedora Infrastructure are the only clients which can really use it. So, RFC1918 addresses should only be entered by admins.
